Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Buttons/Controls Toolkit/Dialog Buttons/KryptonOKDialogButton.cs

```diff
@@ -53,7 +53,10 @@ public KryptonOKDialogButton()
 
         private void KryptonOKDialogButton_TextChanged(object sender, EventArgs e)
         {
-            if (Text == Name) Text = "&OK";
+            if (Text == Name)
+            {
+                Text = "&OK";
```

Comment:
These assignments should be defaulting back to the Global string values

Reply:
Usage of `KryptonManager.Strings`